kafka-client/src/main/java/dev/responsive/db/CassandraFactSchema.java

```diff
@@ -76,30 +73,43 @@ public CassandraFactSchema(final CassandraClient client) {
 
   @Override
   @CheckReturnValue
-  public SimpleStatement create(final String tableName, Optional<Duration> ttl) {
+  public void create(final String tableName, Optional<Duration> ttl) {
     LOG.info("Creating fact data table {} in remote store.", tableName);
-    final CreateTable createTable = SchemaBuilder
+    CreateTableWithOptions createTable = SchemaBuilder
```

Comment:
nit: maybe this is paranoid but I'm a bit nervous about non-final variables like this...could we maybe wrap the shared options in a `createTable()` method and have something like
```
final CreateTable createTable;
if (ttl.isPresent()) {
 createTable = createTable().withTTl(...).withCompaction(...);
} else {
  createTable = createTable().withCompaction(...);
}
```